include/vstwebview/webview_pluginview.h

```diff
@@ -39,7 +39,9 @@ class WebviewPluginView : public Steinberg::Vst::EditorView {
   WebviewPluginView(Steinberg::Vst::EditController *controller,
                     const std::string &title,
                     const std::vector<vstwebview::Bindings *> &bindings,
-                    Steinberg::ViewRect *size = nullptr);
+                    Steinberg::ViewRect *size = nullptr,
+                    const std::string uri = nullptr
+                    );
```

Comment:
nit: ); should probably be after nullptr. Try running through clang-format with --style Google